doc/CONFIGURATION.md

```diff
@@ -319,7 +319,7 @@ See [mounthelper.go](https://github.com/awslabs/mountpoint-s3/tree/main/examples
 
 ## Caching configuration
 
-Mountpoint can optionally cache object metadata and content to reduce cost and improve performance for repeated reads to the same file.
+Mountpoint can optionally cache filesystem metadata and content to reduce cost and improve performance for repeated reads to the same file.
```

Comment:
We're using "file system" instead of "filesystem" throughout, so I'd stick to the former for consistency. (There's a few occurrences to update in this PR.)

Since we need to update the PR anyway, I'd also suggest to keep "object content".

```suggestion
Mountpoint can optionally cache file system metadata and object content to reduce cost and improve performance for repeated reads to the same file.
```